opentreemap/treemap/templates/instance_base.html

```diff
@@ -7,10 +7,13 @@
 
 {% block application_css %}
 
+{% get_files 'js/treemap/base' 'css' as css_files %}
 {% if request.instance.scss_variables %}
-  <link id="application-css" href="{% url 'scss' %}?{{ request.instance.scss_query_string }}" rel="stylesheet">
+<!-- The Webpack bundle name is added as a query parameter for cache busting since
+     it includes a checksum of the bundle, letting us set far-future expires cache headers -->
+  <link id="application-css" rel="stylesheet"
+        href="{% url 'scss' %}?{{ request.instance.scss_query_string }}&url={{ css_files.0.name|urlencode }}">
```

Comment:
I think we're OK on both counts.

The `{% if ... %}` block above checks for `request.instance.scss_variables`, so `scss_query_string` won't be blank.

`css_files.0.` is what we've been using to get the Webpack CSS bundle URL ever since we switched to Webpack, and from inspecting the `webpack-stats.json` file there is only ever one entry per bundle.